components/datadog/agent/host.go

```diff
@@ -77,6 +82,55 @@ func (h *HostAgent) installAgent(env *config.CommonEnvironment, params *agentpar
 		return err
 	}
 
+	varddcmd, err := h.host.OS.Runner().Command(
+		h.namer.ResourceName("var/run/datadog"),
+		&command.Args{
+			Create: pulumi.String("mkdir -p /var/run/datadog"),
+			Sudo:   true,
+		},
+	)
+	if err != nil {
+		return err
+	}
+
+	_, err = h.host.OS.Runner().Command(
+		h.namer.ResourceName("var/run/datadog perm"),
+		&command.Args{
+			Create: pulumi.String("chown dd-agent:dd-agent /var/run/datadog"),
+			Sudo:   true,
+		},
+		utils.PulumiDependsOn(varddcmd),
+		utils.PulumiDependsOn(installCmd),
+	)
+	if err != nil {
+		return err
+	}
+
+	dockerGroup, err := h.host.OS.Runner().Command(
```

Comment:
Those two commands should not be here as it's not related to Agent host install.